data/json/items/gun/shot.json

```diff
@@ -90,13 +78,14 @@
     "name_plural": "Kel-Tec KSG",
     "description": "A bullpup pump-action shotgun, the Kel-Tec KSG uses a pair of magazine tubes to increase its capacity.  Each tube has to be loaded separately, but this offers the option of loading different ammunition for different situations.",
     "weight": "1550 g",
-    "volume": 0,
+    "volume": "4784 ml",
     "price": 99000,
     "to_hit": -1,
     "bashing": 9,
     "material": [ "steel", "plastic" ],
-    "ranged_damage": 3,
-    "dispersion": 345,
+    "ranged_damage": 1,
+    "dispersion": 425,
+    "sight_dispersion": 1000,
```

Comment:
Why a separate sight dispersion that's so high? No default dispersion is above 50 O_o

Reply:
The intent here is that these guns do not come with default sights of any kind.